testing/test_warnings.py

```diff
@@ -106,3 +109,29 @@ def test_ignore(testdir, pyfile_with_warnings, method):
     ])
     assert WARNINGS_SUMMARY_HEADER not in result.stdout.str()
 
+
+
+def test_unicode(testdir, pyfile_with_warnings):
+    testdir.makepyfile('''
```

Comment:
Sorry did not realize we did not prepare the CHANGELOG for new entries.

Just something like this to the CHANGELOG:

```
3.1.1 (unreleased)
==================

* Fix encoding errors for unicode warnings in Python 2.
```

We are introducing towncrier to manage the CHANGELOG in #2431, but we will take care of moving the entry to that format later! 👍